control/src/spi_interface.c

```diff
@@ -0,0 +1,90 @@
+
+
+#include "spi_interface.h"
+
+#include <wiringPiSPI.h>   // only available on Raspberry Pi 
+
+#include <stdio.h>
+#include <unistd.h>
+#include <string.h>
+
+/*************************************************************************************************************************************************
+ * Globals Variables
+ * 
+ * Notes:
+ * Channel is the wiringPi name for the chip select (or chip enable) pin. Both channel 0 and channel 1 will be used.
+ * Will add logic to allow for both later.
+*************************************************************************************************************************************************/
+static const int CHANNEL0 = 0;
+static const int CHANNEL1 = 1;
+
+/*************************************************************************************************************************************************
+ * Function: 
+ * 
+ * Notes:
+ * Channel is the wiringPi name for the chip select (or chip enable) pin. Both channel 0 and channel 1 will be used.
+ * Will add logic to allow for both later.
+*************************************************************************************************************************************************/
+bool spiLogic(char* instruction)
```

Comment:
This function name doesn't convey what the function does. If you intend this library to be specifically used for the encoder buffer, then calling the file spi_interface and the function spiLogic is too nonspecific. It never says even mentions the encoder buffer

Reply:
Found better name